src/core/processing/qgsprocessingparameters.cpp

```diff
@@ -3210,7 +3213,7 @@ QString QgsProcessingParameterGeometry::asScriptCode() const
         break;
 
       default:
-        code += QLatin1String( "unknown" );
+        code += QLatin1String( "unknown " );
```

Comment:
Can this change be reverted ?

Reply:
Why ? this is required to leave a blank after parameter.